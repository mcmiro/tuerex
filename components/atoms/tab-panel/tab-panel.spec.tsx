import * as React from "react";
import {screen, render} from "@testing-library/react";
import TabPanel from "./";

describe("Test TabPanel component", () => {
	it("Should pass when children on TabPanel is defined", () => {
		render(<TabPanel data-testid="tab-panel-test">Test</TabPanel>);

		const tabPanel = screen.queryByTestId("tab-panel-test");
		expect(tabPanel).toBeDefined();
		expect(tabPanel).toMatchSnapshot();
	});
});
