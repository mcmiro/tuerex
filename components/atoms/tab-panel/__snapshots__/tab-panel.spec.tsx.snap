// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TabPanel component Should pass when children on TabPanel is defined 1`] = `null`;
